cinema/serializers.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from .models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=True)
+    description = serializers.CharField(required=True)

```

Comment:
is it possible to add here max_length?

Reply:
dont need, because description = models.**TextField()**